dynamic-graph-service/ci/ci_check.sh

```diff
@@ -0,0 +1,45 @@
+SCRIPT_DIR=$( cd "${0%/*}" && pwd -P )
```

Comment:
Can we put these checks into ci/workflow for both gl and dgs?

Reply:
the ci of dgs is disable now, we can refactor them in later commits.